netbox/utilities/templatetags/helpers.py

```diff
@@ -234,10 +234,19 @@ def utilization_graph(utilization, warning_threshold=75, danger_threshold=90):
     """
     Display a horizontal bar graph indicating a percentage of utilization.
     """
+    if isinstance(utilization, int):
```

Comment:
The function's return signature needs to be consistent and predictable; that is, it should always return the same context. A better approach might be to require the numerator and denominator, and calculate the resulting percentage internally. (Note that this would entail updating all other usages of this function as well.)